packages/ui/src/lib/table/Table.spec.ts

```diff
@@ -380,3 +380,20 @@ test('Expect duration cell to be empty when undefined', async () => {
     expect(cells[6].textContent?.trim()).toBe(expected[i - 1]);
   }
 });
+
+test('Expect table is scoped for css manipulation', async () => {
```

Comment:
nit: the reason for this change is to keep fix table rendering in the case when multiple tables are visible. The test proves table rendering does not conflict with other UI elements. There should be tests for two tables rendered in the same page.

Reply:
I share the same feedback, having a MultipleTableTest.svelte component or something similar

Reply:
test updated